cmd/installer/root.go

```diff
@@ -1075,7 +1078,7 @@ func SetPackRoot(packRoot string, create bool) error {
 		return errs.ErrPackRootDoesNotExist
 	}
 
-	checkConnection := viper.GetBool("check-connection")
+	checkConnection := viper.GetBool("check-connection") // TODO: never set
```

Comment:
What does the TODO has as implications?

Reply:
TODO here means to find the reason why this command line option was there someday but then removed. Also it means to look deeper what this option was thought to do. A quick look does not clearly show what it was good for.